.github/workflows/benchmarks.yml

```diff
@@ -11,7 +11,7 @@ env:
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +6]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Could you send a temporary commit here enabling these on PRs so we can make sure they're passing before merging? Then you can remove the commit before merging. I think something like this here would work:

```gt_suggestion
  pull_request:
    branches:
      - "**"
  workflow_dispatch:
```

Same for the other test below<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/3018?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Ok, done.